lib/src/block_parser.dart

```diff
@@ -114,10 +126,21 @@ class BlockParser {
   BlockSyntax? get parentSyntax => _parentSyntax;
   BlockSyntax? _parentSyntax;
 
+  /// If the [SetextHeadingSyntax] is disabled temporarily
```

Comment:
Start this comment with "Whether", and end with a period.

https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-a-boolean-variable-or-property-comment-with-whether-followed-by-a-noun-or-gerund-phrase

Reply:
Fixed! Thanks!